ev3dev/sound.py

```diff
@@ -75,8 +73,21 @@ class Sound:
 
     channel = None
 
-    @staticmethod
-    def beep(args=''):
+    # play_types
+    PLAY_WAIT_FOR_COMPLETE = 0
+    PLAY_NO_WAIT_FOR_COMPLETE = 1
+    PLAY_LOOP = 2
+
+    PLAY_TYPES = (
+        PLAY_WAIT_FOR_COMPLETE,
+        PLAY_NO_WAIT_FOR_COMPLETE,
+        PLAY_LOOP
+    )
```

Comment:
@WasabiFan  this may answer your question above: @dwalton introduces these parameters, so the `Sound` class is now stateful. But, @dwalton, why are these capitalized? These should be user settings, not constants, right?

Reply:
Ah, sorry. This are not parameters, these are types of operation.